gui/default/syncthing/device/changeLogModalView.html

```diff
@@ -0,0 +1,26 @@
+<style> th, td { padding: 6px; } </style>
+<modal id="globalDiskChanges" status="default" icon="{{'history'}}" heading="{{'Global Changes' | translate}}" large="yes" closeable="yes">
+    <div class="modal-body">
+        <table>
+          <tr>
+            <th translate>Device</th>
+            <th translate>Action</th>
+            <th translate>Type</th>
+            <th translate>Path</th>
+            <th translate>Time</th>
+          </tr>
+          <tr ng-repeat="changeEvent in globalChangeEvents">
+            <td>{{deviceName(findDevice(changeEvent.data.modifiedBy))}}</td>
```

Comment:
(Loop through all known devices. For each one, look at the device ID. If the device ID starts with your short ID, return its name. If none are found, return the short ID. For bonus points, see if more than one device ID has the same prefix, and do something sensible in that case).